### PR TITLE
Play celebratory music 

### LINKED ISSUE #76: [FEATURE] Play longer tune when game ends
Play a longer tune when the game ends

### PR DESCRIPTION
**Pull request by lpatmo**
## Description
This PR resolves #76

## After

Play celebratory music after the end of a game

## Implementation Details 
Re-used the original data so had to .splice(0,6) in a few places to make sure we're still only playing 6 notes in non-celebratory moments

### CODE REVIEW

File: src/Board.js [hunk 1]
```diff
@@ -33,17 +33,17 @@ function Board({ answer }) {
   const handleSubmit = useCallback(
     (e) => {
       e.preventDefault();
-      let answerStr = answer.sequence
+      let answerStr = answer.sequence.slice(0,6)
         .map((noteCluster) => noteCluster.split("")[0])
         .join("");
-
+      console.log('answerStr', answerStr)
```
Comment (arturolei) [inline src/Board.js:39]: Why has the `console.log` been left in?

Comment (arturolei) [inline src/Board.js:39]:
```suggestion
```

**Comment by arturolei** [comment]
Looks okay. Note: might want to add mute button or confetti for celebration.

**Review by ISPOL — APPROVED** [review]
Looks fine to me.

**Comment by lpatmo** [comment]
@arturolei @ISPOL Thanks for reviewing!

**Pull request merged**